internal/storage/storage_handle.go

```diff
@@ -144,6 +153,29 @@ func createHTTPClientHandle(ctx context.Context, clientConfig *storageutil.Stora
 		clientOpts = append(clientOpts, option.WithEndpoint(clientConfig.CustomEndpoint))
 	}
 
+	if clientConfig.ReadStallRetryConfig.Enable {
+		// Hidden way to modify the increase rate for dynamic delay algorithm in go-sdk.
+		// Ref: http://shortn/_417eTbNbKK
+		// Temporarily we kept an option to change the increase-rate, will be removed
+		// once we get a good default.
+		err = os.Setenv(dynamicReadReqIncreaseRateEnv, strconv.FormatFloat(clientConfig.ReadStallRetryConfig.ReqIncreaseRate, 'f', -1, 64))
```

Comment:
Setting environment variables in the middle of the lifetime of a daemon seems like a brittle design. I am assuming that this env var is read by the go sdk. Is it not possible to pass this down to go sdk (or whoever the consumer is) some other way e.g. a function call ?

Reply:
As said in the comment, these are temporary change only for testing and will be removed after finding a good default.